pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest.java

```diff
@@ -615,21 +615,16 @@ public void testInvalidDynamicConfigContentInZK() throws Exception {
      *
      * @throws Exception
      */
-    @Test
+    @Test(timeOut = 5000)
```

Comment:
We don’t need this, Awaitility will time out 10 seconds by default

Reply:
OK~ I will fix it.